cpp/benchmarks/groupby/group_max.cpp

```diff
@@ -91,7 +96,8 @@ NVBENCH_BENCH_TYPES(bench_groupby_max,
   .set_name("groupby_max")
   .add_int64_axis("cardinality", {0})
   .add_int64_power_of_two_axis("num_rows", {12, 18, 24})
-  .add_float64_axis("null_probability", {0, 0.1, 0.9});
+  .add_float64_axis("null_probability", {0, 0.1, 0.9})
+  .add_int64_axis("num_aggregations", {1, 2, 4, 8, 16, 32});
```

Comment:
I would suggest to leave this as 
`.add_int64_axis("num_aggregations", {1});` to prevent the runtime from increasing

Reply:
Updated in 8e08aa4.